README.rst

```diff
@@ -448,15 +448,29 @@ Please make your best effort to comply with `black` and `pylint` before using di
 Authors
 *******
 
-Lightning is the work of `many contributors <https://github.com/PennyLaneAI/pennylane-lightning/graphs/contributors>`_.
-
-If you are doing research using PennyLane and Lightning, please cite `our paper <https://arxiv.org/abs/1811.04968>`_:
+.. citation-start-inclusion-marker-do-not-remove
 
-    Ville Bergholm, Josh Izaac, Maria Schuld, Christian Gogolin, M. Sohaib Alam, Shahnawaz Ahmed,
-    Juan Miguel Arrazola, Carsten Blank, Alain Delgado, Soran Jahangiri, Keri McKiernan, Johannes Jakob Meyer,
-    Zeyue Niu, Antal Száva, and Nathan Killoran.
-    *PennyLane: Automatic differentiation of hybrid quantum-classical computations.* 2018. arXiv:1811.04968
+Lightning is the work of `many contributors <https://github.com/PennyLaneAI/pennylane-lightning/graphs/contributors>`_.
 
+If you are using PennyLane and Lightning for research, please cite `[1] <https://arxiv.org/pdf/2403.02512>`_:
+
+    [1] Ali Asadi, Amintor Dusko, Chae-Yeun Park, Vincent Michaud-Rioux, Isidor Schoch, Shuli Shu, 
+    Trevor Vincent, and Lee James O'Riordan.
+    *Hybrid quantum programming with PennyLane Lightning on HPC platforms.* 2024. `arXiv:2403.02512 <https://arxiv.org/abs/2403.02512>`_
+
+```bibtex
+    @misc{asadi2024hybridquantumprogrammingpennylane,
+        title={{Hybrid quantum programming with PennyLane Lightning on HPC platforms}}, 
+        author={Ali Asadi and Amintor Dusko and Chae-Yeun Park and Vincent Michaud-Rioux and Isidor Schoch and Shuli Shu and Trevor Vincent and Lee James O'Riordan},
+        year={2024},
+        eprint={2403.02512},
+        archivePrefix={arXiv},
+        primaryClass={quant-ph},
+        url={https://arxiv.org/abs/2403.02512}, 
+    }
+```
```

Comment:
````suggestion
```bibtex
@misc{asadi2024hybridquantumprogrammingpennylane,
    title={{Hybrid quantum programming with PennyLane Lightning on HPC platforms}}, 
    author={Ali Asadi and Amintor Dusko and Chae-Yeun Park and Vincent Michaud-Rioux and Isidor Schoch and Shuli Shu and Trevor Vincent and Lee James O'Riordan},
    year={2024},
    eprint={2403.02512},
    archivePrefix={arXiv},
    primaryClass={quant-ph},
    url={https://arxiv.org/abs/2403.02512}, 
}
```
````

Sorry, trying to figure out why this is not rendering correctly 🤔 Maybe it's indentation

Reply:
<img width="1014" alt="image" src="https://github.com/user-attachments/assets/1d407af4-96cd-4488-a922-0a63eda9786c">
 this is how it currently looks

Reply:
RST README vs markdown readme? Maybe the backticks aren't expected in this format

Reply:
🤦🤦🤦🤦🤦🤦🤦 my bad. It's because PL/catalyst both have MD readmes!

Reply:
```suggestion
.. code-block:: bibtex

    @misc{asadi2024hybridquantumprogrammingpennylane,
        title={{Hybrid quantum programming with PennyLane Lightning on HPC platforms}}, 
        author={Ali Asadi and Amintor Dusko and Chae-Yeun Park and Vincent Michaud-Rioux and Isidor Schoch and Shuli Shu and Trevor Vincent and Lee James O'Riordan},
        year={2024},
        eprint={2403.02512},
        archivePrefix={arXiv},
        primaryClass={quant-ph},
        url={https://arxiv.org/abs/2403.02512}, 
    }
```

Reply:
When RTD builds, give it one final inspection?

Reply:
Looks good!